concepts/basics/introduction.md

```diff
@@ -0,0 +1,111 @@
+# Introduction
```

Comment:
I think this is a very solid introduction, I think a mention of `;` could be valuable, since most languages don't have it / require it. I think it should not be that long but more or saying it has to be after each line could be helpful.

Reply:
You're right. I think I had it in there once and decided on "show don't tell" because there are so many exceptions to any rule I tried to state. If someone has a good sentence, I'm all in.